balancer/frontend-v2
# Hotfix: ignore gearbox linear pool type

# Description

This is a linear pool that shouldn't be shown in the pools list. 

## Type of change


## How should this be tested?

- Search for dDAI on Ethereum
- Ensure there are no linear pools in the list (list should be empty currently)

## Checklist:
